storm-core/src/clj/backtype/storm/command/activate.clj

```diff
@@ -14,11 +14,11 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 (ns backtype.storm.command.activate
-  (:use [backtype.storm thrift log])
-  (:gen-class))
+    (:require [backtype.storm.thrift :as t]
+      [backtype.storm.log :as log])
```

Comment:
Indentation is a little off here, and I'd prefer `:as thrift` or somesuch rather than `:as t`.